client/checkout/classic/event-handlers.js

```diff
@@ -251,71 +240,15 @@ jQuery( function ( $ ) {
 		if ( hasPaymentMethodCountryRestrictions( upeElement ) ) {
 			togglePaymentMethodForCountry( upeElement );
 
-			// this event only applies to the checkout form, but not "place order" or "add payment method" pages.
-			$( '#billing_country' ).on( 'change', function () {
-				togglePaymentMethodForCountry( upeElement );
-			} );
-		}
-	}
-
-	function isBillingInformationMissing() {
-		const enabledBillingFields = getUPEConfig( 'enabledBillingFields' );
-
-		// first name and last name are kinda special - we just need one of them to be at checkout
-		const name = `${
-			document.querySelector(
-				`#${ SHORTCODE_BILLING_ADDRESS_FIELDS.first_name }`
-			)?.value || ''
-		} ${
-			document.querySelector(
-				`#${ SHORTCODE_BILLING_ADDRESS_FIELDS.last_name }`
-			)?.value || ''
-		}`.trim();
-		if (
-			! name &&
-			( enabledBillingFields[
-				SHORTCODE_BILLING_ADDRESS_FIELDS.first_name
-			] ||
-				enabledBillingFields[
-					SHORTCODE_BILLING_ADDRESS_FIELDS.last_name
-				] )
-		) {
-			return true;
+			const billingInput = upeElement
+				?.closest( 'form.checkout' )
+				?.querySelector( '[name="billing_country"]' );
```

Comment:
```suggestion
				?.querySelector(
					`[name="${ SHORTCODE_BILLING_ADDRESS_FIELDS.country }"]`
				);
```

Very minor suggestion, since I noticed that the previous changes were making references to the `SHORTCODE_BILLING_ADDRESS_FIELDS` constant